Simplenote/Base.lproj/MainMenu.xib

```diff
@@ -98,28 +88,9 @@
                                     <action selector="emptyTrashWasPressed:" target="494" id="hK4-FN-3lK"/>
                                 </connections>
                             </menuItem>
-                            <menuItem isSeparatorItem="YES" id="Jd8-h6-yd1"/>
-                            <menuItem title="Log Out" id="Pde-gm-aAD">
-                                <modifierMask key="keyEquivalentModifierMask"/>
-                                <attributedString key="userComments">
-                                    <fragment content="Logs out from the Account"/>
-                                </attributedString>
-                                <connections>
-                                    <action selector="signOutAction:" target="494" id="VGf-Gg-vor"/>
-                                </connections>
-                            </menuItem>
                             <menuItem isSeparatorItem="YES" id="1548">
                                 <modifierMask key="keyEquivalentModifierMask" command="YES"/>
                             </menuItem>
-                            <menuItem title="Delete Account" id="1sa-fK-USq">
```

Comment:
I'm not sure if this PR is needed still, since this menuItem (among others) were already dropped [here](https://github.com/Automattic/simplenote-macos/pull/1045/files#diff-8990f169754cc4f51b45d79cbce55a366ae4f45066b49b542c71dea35441186eL106) ?